plugins/modules/apache2_mod_proxy.py

```diff
@@ -116,19 +116,20 @@
   description: Specific balancer member information dictionary, returned when the module is invoked with O(member_host) parameter.
   type: dict
   returned: success
-  sample:
-    {"attributes":
-          {"Busy": "0",
-          "Elected": "42",
-          "Factor": "1",
-          "From": "136K",
-          "Load": "0",
-          "Route": null,
-          "RouteRedir": null,
-          "Set": "0",
-          "Status": "Init Ok ",
-          "To": " 47K",
-          "Worker URL": null
+  sample: |
```

Comment:
Since the return type is dictionary, this should not be a string, but a dictionary (as it was before).

Reply:
Fixed. 

I have been working on andebox these last couple of days. I have removed the generation of `|` from that code. With that, we should be able to format the JSON blobs within the YAML blocks within the plugins. This module was kind of a test drive.